docs/core/policies.rst

```diff
@@ -344,19 +348,85 @@ It is recommended to use
         ``batch_size`` is required, pass an ``int``, e.g.
         ``"batch_size": 8``.
 
-    These parameters can be specified in the policy configuration file.
-    The default values are defined in ``EmbeddingPolicy.defaults``:
+    These parameters can be specified in the configuration file.
+    The following default values are set:
+
+    .. code-block:: yaml
 
-    .. literalinclude:: ../../rasa/core/policies/embedding_policy.py
-       :dedent: 4
-       :start-after: # default properties (DOC MARKER - don't remove)
-       :end-before: # end default properties (DOC MARKER - don't remove)
+        # nn architecture
+        # a list of hidden layers sizes before dialogue and action embed layers
```

Comment:
?
```suggestion
        # a list of hidden layers sizes before dialogue and action embedding layers
```

Reply:
if this is copied from the code then we should do this code include tags instead of duplicating it.

Reply:
We are using constants in code. They are not replaced when included here via the code include tags. As the constant names might be different from the values, we add the defaults manually. Let me know if you know a better way on how to do this.